packages/terra-site/src/examples/action-footer/examples/centered-action-footer/MultipleActions.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import CenteredActionFooter from 'terra-action-footer/lib/CenteredActionFooter';
+import Button from 'terra-button';
+import Spacer from 'terra-spacer';
+
+const MultipleActionsExample = () => (
+  <CenteredActionFooter
+    center={
+      <div>
```

Comment:
UX Edits (1 of 3): 
This layout of multiple actions inline would never be recommended to be used in actual practice, so lets change it to look closer to actual usage: 
- two default (neutral) style buttons, stack vertically with the `isBlock ` 
- text can read "First Action" and "Second Action" and make them
- add inline style to make the containing div have a max-width of 400px

Reply:
Just to clarify, without using an inline style for the containing div for these vertically stacked buttons, is this fine?
![screen shot 2018-03-13 at 4 55 34 pm](https://user-images.githubusercontent.com/34115417/37372083-91ad982e-26df-11e8-84d0-97c4d89121ef.png)

(will refactor to accomodate [your third edit below](https://github.com/cerner/terra-core/pull/1329#issuecomment-372768345)) Or are you wanting them to fill the width of the footer?